xml/nsx-deploy-os-cobbler.xml

```diff
@@ -31,9 +31,9 @@
     the &o_comp; ESX compute proxies are VMs. Controllers 1, 2, and 3 are
     running on physical servers.
    </para>
-   <screen>&prompt.ardana;cd ~/openstack/ardana/ansible
-   &prompt.ardana;ansible-playbook -i hosts/localhost bm-reimage.yml -e
-   nodelist=controller1,controller2,controller3</screen>
+   <screen><?dbsuse-fo font-size="0.70em"?>
+&prompt.ardana;cd ~/openstack/ardana/ansible
+&prompt.ardana;ansible-playbook -i hosts/localhost bm-reimage.yml -e nodelist=controller1,controller2,controller3</screen>
```

Comment:
Rather than adding the `<?dbsuse-fo` PI, maybe keep this as too lines (but with a trailing `\`!)